server/constants/paymentMethods.ts

```diff
@@ -17,7 +17,7 @@ export enum PAYMENT_METHOD_TYPE {
   COLLECTIVE = 'collective',
   HOST = 'host',
   ADAPTIVE = 'adaptive',
-  GIFT_CARD = 'giftcard',
```

Comment:
I would tend to the opposite direction: all entries should have this`_` (eg. `CREDIT_CARD`) as that's how things are formatted in our other enums (snake uppercase).

Reply:
I'm favoring consistency with the database values for now.

I get that constants are effectively here to help having nicer/different values than the database one.

But as we already suffered too much on this topic, I think we need to relax a bit with a simple mental model.